### PR TITLE
[PWN-6853] Form redesign

### PR DESCRIPTION
**Pull request by sidorov-panda**
## Link jira to issue
https://p2pvalidator.atlassian.net/browse/PWN-6853

## Description of the changes
-

### CODE REVIEW

File: p2p_wallet/Scenes/Main/Sell/SellInputView/SellInputView.swift [hunk 1]
```diff
@@ -58,68 +54,90 @@ struct SellInputView: View {
...
+                        textField.maximumFractionDigits = 2
+                        textField.decimalSeparator = "."
+                        textField.textAlignment = .right
+                    }
+                } else {
+                    DecimalTextField(
+                         value: $viewModel.quoteAmount,
+                         isFirstResponder: $viewModel.isEnteringQuoteAmount
+                     ) { textField in
+                         textField.font = UIFont.font(of: .largeTitle, weight: .regular)
```
Comment (qwerkly) [inline p2p_wallet/Scenes/Main/Sell/SellInputView/SellInputView.swift:87]: `UIFont.font(of: .largeTitle, weight: .regular)` change to `.font(of: .largeTitle, weight: .regular)`

File: p2p_wallet/Scenes/Main/Sell/SellInputView/SellInputView.swift [hunk 2]
```diff
@@ -58,68 +54,90 @@ struct SellInputView: View {
...
+                   .foregroundColor(Color(Asset.Colors.night.color))
+
+                Spacer()
+
+                if viewModel.showingBaseAmount {
+                    DecimalTextField(
+                        value: $viewModel.baseAmount,
+                        isFirstResponder: $viewModel.isEnteringBaseAmount
+                    ) { textField in
+                        textField.font = UIFont.font(of: .largeTitle, weight: .regular)
```
Comment (qwerkly) [inline p2p_wallet/Scenes/Main/Sell/SellInputView/SellInputView.swift:76]: `UIFont.font(of: .largeTitle, weight: .regular)` change to `.font(of: .largeTitle, weight: .regular)`

File: p2p_wallet/Scenes/Main/Sell/SellInputView/SellInputView.swift [hunk 3]
```diff
@@ -58,68 +54,90 @@ struct SellInputView: View {
...
+                         value: $viewModel.quoteAmount,
+                         isFirstResponder: $viewModel.isEnteringQuoteAmount
+                     ) { textField in
+                         textField.font = UIFont.font(of: .largeTitle, weight: .regular)
+                         textField.keyboardType = .decimalPad
+                         textField.maximumFractionDigits = 2
+                         textField.decimalSeparator = "."
+                         textField.textAlignment = .right
+                     }
+                }
```
Comment (qwerkly) [inline p2p_wallet/Scenes/Main/Sell/SellInputView/SellInputView.swift:93]: Copy paste.
Create func, that sets parameters for textField and return it

File: p2p_wallet/Scenes/Main/Sell/SellViewModel.swift [hunk 4]
```diff
@@ -57,17 +58,23 @@ class SellViewModel: BaseViewModel, ObservableObject {
     @Published var baseCurrencyCode: String = "SOL"
     @Published var baseAmount: Double?
     @Published var maxBaseAmount: Double?
+    /// Mostly used to show keyboard
     @Published var isEnteringBaseAmount: Bool = true
+    @Published var isEnteringQuoteAmount: Bool = false
```
Comment (qwerkly) [inline p2p_wallet/Scenes/Main/Sell/SellViewModel.swift:63]: Remove `: Bool`

File: p2p_wallet/Scenes/Main/Sell/SellInputView/SellInputView.swift [hunk 5]
```diff
@@ -142,38 +142,42 @@ struct SellInputView: View {
...
                     .descriptionTextStyle()
                     .skeleton(
                         with: true,
                         size: CGSize(width: 126, height: 16),
                         animated: .default
                     )
             case .loaded(let fee):
-                Text(L10n.includedFee("\(fee.toString()) \(viewModel.quoteCurrencyCode)"))
-                    .descriptionTextStyle()
+                Text("All fees included \(fee.baseAmount.toString(maximumFractionDigits: 2)) \(viewModel.baseCurrencyCode) ≈ \(fee.quoteAmount.toString()) \(viewModel.quoteCurrencyCode)")
```
Comment (sidorov-panda) [inline p2p_wallet/Scenes/Main/Sell/SellInputView/SellInputView.swift:164]: Localisation

File: p2p_wallet/Scenes/Main/Sell/SellInputView/SellInputView.swift [hunk 6]
```diff
@@ -58,68 +54,72 @@ struct SellInputView: View {
...
+        .blockStyle()
         .padding(.top, 44)
     }
 
+    var switchButton: some View {
+        Button(action: {
+            viewModel.showingBaseAmount.toggle()
+        }) {
+            Text(
+                "Switch to " + (viewModel.showingBaseAmount ? viewModel.quoteCurrencyCode : viewModel.baseCurrencyCode)
```
Comment (sidorov-panda) [inline p2p_wallet/Scenes/Main/Sell/SellInputView/SellInputView.swift:98]: Localization

File: p2p_wallet/Scenes/Main/Sell/SellInputView/SellInputView.swift [hunk 7]
```diff
@@ -58,68 +54,72 @@ struct SellInputView: View {
...
-                     textField.font = UIFont.font(of: .text3, weight: .regular)
-                     textField.keyboardType = .decimalPad
-                     textField.maximumFractionDigits = 2
-                     textField.decimalSeparator = "."
-                 }
-
-                 Text(viewModel.baseCurrencyCode)
-                    .apply(style: .text3)
-                    .foregroundColor(Color(Asset.Colors.night.color).opacity(0.3))
+                Text("Cash out \(viewModel.baseCurrencyCode), receive \(viewModel.quoteCurrencyCode)")
```
Comment (sidorov-panda) [inline p2p_wallet/Scenes/Main/Sell/SellInputView/SellInputView.swift:79]: localisation

**Pull request merged**